test/DataQI.Commons.Test/Repository/Sample/CustomFakeRepository.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DataQI.Commons.Test.Repository.Sample
+{
+    public class CustomFakeRepository
+    {
+        private readonly IEnumerable<FakeEntity> fakeEntities;
+
+        public CustomFakeRepository() : this(Array.Empty<FakeEntity>())
```

Comment:
What do you think of using `Enumerable.Empty<FakeEntity>()`?

Reply:
Yeah ;-)